package.json

```diff
@@ -54,5 +55,12 @@
   "devDependencies": {
     "jest": "26.6.3",
     "npm-run-all": "4.1.5"
+  },
+  "peerDependencies": {
+    "react-native": "*",
+    "fast-sha256": "^1.3.0",
+    "jssha": "^3.2.0",
+    "react-native-get-random-values": "^1.7.0",
+    "text-encoding": "^0.7.0"
   }
```

Comment:
The package file.json is used for all environments, so I think it's better to remove peerDependencies for react-native from here. Instead, I will add instructions for installing these dependencies to the documentation